multimodal/src/autogluon/multimodal/utils/export.py

```diff
@@ -130,13 +131,14 @@ def export_onnx(
         input_keys = self._model.input_keys
         input_vec = [batch[k] for k in input_keys]
 
-        # Infer default onnx path, and create parent directory if needed
+        # Write to BytesIO if path argument is not provided
```

Comment:
Let's add some docstring on why we decided to do so as it is non-typical to write to BytesIO if no path is provided. Generally this would mean write to the current folder.